amnesia/src/CMakeLists.txt

```diff
@@ -8,71 +8,71 @@ option(USERDIR_RESOURCES "Allow using local userdirectory resources" ON)
 option(FBX_SUPPORT "Add FBX support during compilation" OFF)
 
 set(CMAKE_BUILD_TYPE "Release")
-# Release: high optimization level, no debug info, code or asserts.
-# Debug: No optimization, asserts enabled, debug info included in executable (so you can step through the code with a debugger and have address to source-file:line-number translation).
-# RelWithDebInfo: optimized, *with* debug info, but no debug (output) code or asserts.
-# MinSizeRel: same as Release but optimizing for size rather than speed.
+# Release: High optimization level, no debug info, no debug code, no asserts.
+# Debug: No optimization, asserts enabled, debug info included (enables stepping through code with a debugger and source-file:line-number translation).
+# RelWithDebInfo: Optimized with debug info, but no debug output code or asserts.
+# MinSizeRel: Same as Release but optimized for size rather than speed.
 
 set(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR}/../output)
 
 if(MINGW)
-    # There is some stuff that breaks in mingw in newer standards
+    # There are some issues that break in MinGW with newer standards.
```

Comment:
If we're being pedantic about grammar, it's not issues which break

Reply:
Done.